string model1
string link1
string collision1
string model2
string link2
string collision2
---

